src/app/content/highlights/components/EditCard.tsx

```diff
@@ -61,13 +61,24 @@ const EditCard = React.forwardRef<HTMLElement, Props>((
   const trackCreateNote = useAnalyticsEvent('createNote');
   const trackEditNoteColor = useAnalyticsEvent('editNoteColor');
   const trackEditAnnotation = useAnalyticsEvent('editAnnotation');
+  const trackShowCreate = useAnalyticsEvent('showCreate');
+  const trackShowLogin = useAnalyticsEvent('showLogin');
 
   const blurIfNotEditing = () => {
     if (!editingAnnotation) {
       onBlur();
     }
   };
 
+  React.useEffect(() => {
+    if (authenticated) {
+      trackShowCreate();
+    } else {
+      trackShowLogin();
+    }
```

Comment:
isn't this going to trigger `trackShowCreate` on page load for every highlight that already exists?